db/migrate/20210408145903_create_fields.rb

```diff
@@ -0,0 +1,19 @@
+class CreateFields < ActiveRecord::Migration[6.1]
+  def change
+    create_table :fields do |t|
+      t.uuid :uuid, null: false
+      t.references :calculator
```

Comment:
This field as well as `selector`, `type`, and `kind` should have `NOT NULL` constraint

Reply:
Ok. I made this changes. I'm not sure if I'm doing in correct way every time using "rebase -i" command.